### DISCUSSION TITLE
Mandatory fields only

### DISCUSSION THREAD

**Discussion by joeken** [kimai/kimai#2256] [Questions & Answers]
Is there a possibility to configure kimai2 in the manner, that onle mandatory fields are shown and allowed to edit by usual users in the time records?
I'll know from much eralier versions, that it was possible to hide the description in the records.

**Comment by kevinpapst** [comment]
This feature will not be integrated into Kimai anytime soon.
But you can install the CustomCSSBundle and write the required CSS rules to hide the fields.

**Comment by joeken** [comment]
Thanks for your replay. I tried it and we can solve it with the CustomCSSBundle :)
